sdk/storage/storage-blob/review/storage-blob.api.ts

```diff
@@ -0,0 +1,1424 @@
+import * as Models from "./generated/src/models";
+
+// @public
+class AccountSASPermissions {
+  add: boolean;
+  create: boolean;
+  delete: boolean;
+  list: boolean;
+  static parse(permissions: string): AccountSASPermissions;
+  process: boolean;
+  read: boolean;
+  toString(): string;
+  update: boolean;
+  write: boolean;
+}
+
+// @public
+class AccountSASResourceTypes {
+  container: boolean;
+  object: boolean;
+  static parse(resourceTypes: string): AccountSASResourceTypes;
+  service: boolean;
+  toString(): string;
+}
+
+// @public
+class AccountSASServices {
+  blob: boolean;
+  file: boolean;
+  static parse(services: string): AccountSASServices;
+  queue: boolean;
+  table: boolean;
+  toString(): string;
+}
+
+// @public
+interface AccountSASSignatureValues {
+  expiryTime: Date;
+  ipRange?: IPRange;
+  permissions: string;
+  protocol?: SASProtocol;
+  resourceTypes: string;
+  services: string;
+  startTime?: Date;
+  version?: string;
+}
+
+// @public
+class AnonymousCredential extends Credential {
+  create(nextPolicy: RequestPolicy, options: RequestPolicyOptions): AnonymousCredentialPolicy;
+}
+
+// @public
+class AnonymousCredentialPolicy extends CredentialPolicy {
+  constructor(nextPolicy: RequestPolicy, options: RequestPolicyOptions);
+}
+
+// @public (undocumented)
+interface AppendBlobAppendBlockFromURLOptions {
+  abortSignal?: AbortSignalLike;
+  accessConditions?: AppendBlobAccessConditions;
+  customerProvidedKey?: Models.CpkInfo;
+  sourceContentCrc64?: Uint8Array;
+  sourceContentMD5?: Uint8Array;
+  sourceModifiedAccessConditions?: Models.ModifiedAccessConditions;
+}
+
+// @public
+interface AppendBlobAppendBlockOptions {
+  abortSignal?: AbortSignalLike;
+  accessConditions?: AppendBlobAccessConditions;
+  customerProvidedKey?: Models.CpkInfo;
+  progress?: (progress: TransferProgressEvent) => void;
+  transactionalContentCrc64?: Uint8Array;
+  transactionalContentMD5?: Uint8Array;
+}
+
+// @public
+class AppendBlobClient extends BlobClient {
+  constructor(connectionString: string, containerName: string, blobName: string, options?: NewPipelineOptions);
+  // WARNING: The type "Models.AppendBlobAppendBlockResponse" needs to be exported by the package (e.g. added to index.ts)
```

Comment:
I'm assuming all these warnings will go away in the real client?

Reply:
Yeah, it's an API extractor artifact.